io/jvm/src/main/scala/fs2/io/net/unixsocket/JdkUnixSockets.scala

```diff
@@ -42,27 +44,28 @@ private[unixsocket] class JdkUnixSocketsImpl[F[_]: Files](implicit F: Async[F])
     extends UnixSockets.AsyncUnixSockets[F] {
   protected def openChannel(address: UnixSocketAddress) =
     Resource
-      .make(F.blocking(SocketChannel.open(StandardProtocolFamily.UNIX)))(ch =>
-        F.blocking(ch.close())
+      .make(F.blocking(SocketChannel.open(StandardProtocolFamily.UNIX)).pipe(evalOnVirtualThreadIfAvailable(_)))(ch =>
```

Comment:
I think it is more readable when using `pipe`. Using regular function application it obscures the actual code.